Jenkinsfile

```diff
@@ -738,7 +738,9 @@ def rsyncResults(stage) {
     }
 }
 
-def checkLastImage() {
+def checkLastImage(stage) {
```

Comment:
I like it.  Sounds like it can be done as the massive refactoring. 
Libraries , sanitizing messages, etc.